docs/osf-trivia-list/capsule-update.md

```diff
@@ -0,0 +1,42 @@
+# Frequenty Asked Questions about Capsule Update
+
+## Q: Why was the traditional flashrom-based method replaced by capsule-based one?
+
+A: There are many advantages of capsule-based update.
+First of all, it's much easier[1]:
+
+The steps to update efi firmware are:
+
+1. `cat firmware.cap > /dev/efi_capsule_loader`
+2. `reboot`
+
+Not only is it simpler but also more efficient, because there is no need for
+manual reset and disabling all of the protections before update, thus the whole
+process takes less time. Speaking of protection the last but not least reason is
+safety, the signed coalesced data can only be accessed after signatures are
+verified by edk2, ensuring no unauthorized activity is allowed.
+
+## Q: Is fwupd still Linux only?
+
+A: Yes, fwupd is Linux only. However, this isn't the only software that
```

Comment:
```suggestion
A: No, fwupd isn't Linux only. Even the oldest release on GitHub (Feb 3, 2020) has [fwupd-1.3.7-setup-x86_64.exe](https://github.com/fwupd/fwupd/releases/tag/1.3.7). We also added support for BSD systems as part of [fwdup-BSD](https://nlnet.nl/project/fwdup-BSD/) project.

However, fwupd isn't the only software that
```

Please also rewrap it as needed.

Reply:
here's the updated and squashed version: https://github.com/Dasharo/docs/pull/879/commits/e191a0608caba0d2635516a57efc0c3ab5811d8b

Reply:
@wiktormowinski You can always commit suggestions using the `Commit suggestion` button, or if there are multiple, use  `Add suggestion to batch`, and apply them together. This way, the threads will be resolved automatically without needing to do it manually.

See: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

@krystian-hebel Can you please fix this part and merge the PR?

Reply:
@BeataZdunczyk https://github.com/Dasharo/docs/pull/879#discussion_r1721700040 that's what i did last time :c

Reply:
You missed the empty line. Fixed now, as soon as CI finishes I'll merge this PR.